x-pack/plugins/cases/public/components/files/file_type.tsx

```diff
@@ -4,48 +4,43 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import React from 'react';
+import React, { Suspense, lazy } from 'react';
 
+import { EuiLoadingSpinner } from '@elastic/eui';
 import type {
   ExternalReferenceAttachmentType,
   ExternalReferenceAttachmentViewProps,
 } from '../../client/attachment_framework/types';
-import type { DownloadableFile } from './types';
 
 import { AttachmentActionType } from '../../client/attachment_framework/types';
 import { FILE_ATTACHMENT_TYPE } from '../../../common/api';
-import { FileDownloadButton } from './file_download_button';
-import { FileNameLink } from './file_name_link';
-import { FilePreview } from './file_preview';
 import * as i18n from './translations';
 import { isImage, isValidFileExternalReferenceMetadata } from './utils';
-import { useFilePreview } from './use_file_preview';
-import { FileDeleteButton } from './file_delete_button';
 
-interface FileAttachmentEventProps {
-  file: DownloadableFile;
-}
-
-const FileAttachmentEvent = ({ file }: FileAttachmentEventProps) => {
-  const { isPreviewVisible, showPreview, closePreview } = useFilePreview();
+const FileAttachmentEvent = lazy(() =>
```

Comment:
Nice approach 👍

Reply:
This was effective because this attachment type is registered in `x-pack/plugins/cases/public/plugin.ts`.

Before realizing I was trying to lazy load everything everywhere but it didn't work :D until I realized that the cases app is already mostly lazy loaded, these were the exceptions.